astropy/wcs/wcs.py

```diff
@@ -1037,16 +1037,20 @@ def _read_sip_kw(self, header):
             ctype=[header['CTYPE{0}'.format(nax)] for nax in range(1, self.naxis + 1)]
             if any([ctyp[-4 :] != '-SIP' for ctyp in ctype]):
                 message = """
-                Inconsistent SIP distortion information is present in header:
-                SIP coefficients were detected, but CTYPE is missing "-SIP" suffix,
+                Inconsistent SIP distortion information is present in the FITS header and the WCS object:
+                SIP coefficients were detected, but CTYPE is missing a "-SIP" suffix.
+                astropy.wcs is using the SIP distortion coefficients,
                 therefore the coordinates calculated here might be incorrect.
```

Comment:
I would change last part to: ..., therefore computed coordinates may be different from expected values.